bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+| thermostatplus | Thing  | This is for the MAX! Heating Thermostat+. This is the type that can hold the program by itself.                    |
+| wallthermostat | Thing  | MAX! Wall Thermostat.                                                                                              |
+| ecoswitch      | Thing  | MAX! Ecoswitch.                                                                                                    |
+| shuttercontact | Thing  | MAX! Shuttercontact / Window Contact.                                                                              |
+
+Generally one does not have to worry about the thing types as they are automatically defined.
+If for any reason you need to manually define the Things and you are not exactly sure what type of thermostat you have, you can choose `thermostat` for both the thermostat and thermostat+, this will not affect their working.
+
+## Discovery
+
+Start pairing at your MAX! device. It will show up in the inbox and you can it to your configuration. Serial number and rfAddress is set already
+After adding too your configuration you have to set channel pair_mode to ON on your maxcul_bridge and restart
+the pairing on your MAX! device to start the pairing sequence. 
+If the device is already paired and starts re-pairing e.g. after battery change, the state of channel pair_mode is ignored.
+
+## Binding Configuration
+
+There are no binding wide settings as all configuration settings.
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openhab.org) for the latest information.
+
+## Thing Configuration
+
+### CUL Bridge Configuration
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| device   |         |   Yes    | in the form `<device>`, where `<device>` is a local serial port eg `/dev/ttyACM0` |
+| baudrate |         |   No     | one of 75, 110, 300, 1200, 2400, 4800, 9600, 19200, **38400**, 57600, 115200 |
+| parity   |         |   No     | one of EVEN, ODD, MARK, **NONE**, SPACE |
+
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openhab.org) for the latest information.
+
+### CUN Bridge Configuration
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| networkPath   |         |   Yes    | in the form `<device>`, where `<device>` is `<host>:<port>`, where `<host>` is the host name or IP address and `<port>` is the port number. If no `<port>` is provided the default `2323` is assumed |
+
+### MAX! CUL Bridge Configuration
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| timezone | Europe/London | No | set timezone you want the units to be set to |

```

Comment:
Are you aware that you can retrieve the time zone, the user configured by injecting `TimeZoneProvider` e.g. into your handler factory? See https://github.com/openhab/openhab-addons/search?q=TimeZoneProvider for example

Reply:
Using `TimeZoneProvider` sounds like a better solution then old configuration in OH1.